pages/data-providers/[data-provider-id]/overview.jsx

```diff
@@ -12,18 +12,41 @@ import PerformanceChart from 'components/performance-chart'
 // eslint-disable-next-line
 import styles from './overview.css'
 
+const OverviewCard = ({ children, className, ...passProps }) => (
+  <Card
+    className={classNames.use(styles.overviewCard, className)}
+    {...passProps}
+  >
+    {children}
+  </Card>
+)
+
+const BrowseButton = ({
```

Comment:
It's not `BrowseButton` is it? It's more `LinkButton`, `AnchorButton` or `ButtonLink`, all you do here is changing `tag` to `a`. However, you do not even require `href`

Reply:
I took the name from the text which this button always contains. But I will change it to LinkButoon.